docs/sources-asciidoc/src/main/asciidoc/uml/replication-seq-diagram.plantuml

```diff
@@ -0,0 +1,32 @@
+@startuml
+actor UA1
+control SipServlet
+actor UA2
+
+UA1 -> SipServlet: INVITE             
+hnote over SipServlet: 1
+UA1 <- SipServlet: Trying
+hnote over SipServlet: 2
+UA2 <- SipServlet: INVITE
+hnote over SipServlet: 3
+UA2 -> SipServlet: Ringing
+hnote over SipServlet: 4
+UA1 <- SipServlet: Ringing
+hnote over SipServlet: 5
```

Comment:
Please add note on earlyDialog replciation as explained in http://plantuml.com/sequence-diagram
note left
	On EARLY_DIALOG mode, dialog/sipSession will be replicated
end note